data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3057,6 +3057,50 @@ spec:
                       use for instances in this cluster. Deprecated: use FlavorName
                       in DefaultMachinePlatform to define default flavor.'
                     type: string
+                  controlPlanePort:
+                    description: ControlPlanePort contains the control plane port
```

Comment:
Nit: ControlPlanePort contains details of the network attached to the control plane port, with the network either containing one openstack subnet for IPv4 or two openstack subnets for dualstack clusters. Providing this configuration will prevent OpenShift from managing or updating this network and its subnets. The network and its subnets will be used to during creation of all nodes. This is a TechPreview feature and requires setting featureSet to TechPreviewNoUpgrade.

Can we modify this to something like the above? I don't think we want to call out the specific OpenShift component (Installer) in this user facing description.